stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -1771,6 +1892,20 @@ func _TFCOpSetAttrString(_ op: CTFEOp,
   }
 }
 
+/// Wrapper around TFE_OpSetAttrString that handles converting the Swift Stdlib
+/// String into a buffer that TFE_OpSetAttrString can read.
+@usableFromInline
+@_silgen_name("_swift_tfc_OpSetAttrFunctionName")
+func _TFCOpSetAttrFunctionName(_ op: CTFEOp,
+                               _ attrName: UnsafePointer<Int8>,
+                               _ value: String) {
```

Comment:
Are you using this anywhere in IRGen? Plus, since this is to interface C, make it take a C string directly. Rename "value" to "funcName" for clarity.

```suggestion
                               _ funcName: UnsafePointer<Int8>) {
```

Reply:
Yes, this is being used in irgen.